homeassistant/components/myq/cover.py

```diff
@@ -29,36 +31,24 @@
 }
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_TYPE): cv.string,
```

Comment:
It's a breaking change because you have removed the TYPE parameter.

Reply:
But removing it doesn't break anything... If the user keeps that parameter in their config, nothing adverse will happen. Perhaps my definition is wrong: is _any_ change to the schema, regardless of impact, automatically a breaking change?

Reply:
Ah, I think I understand... This means that when people upgrade, they'll now have a key that isn't accepted in the schema (regardless of what that key actually does). I'll ignore it for now.

Reply:
How do you feel about what I did in cabf89a?